vegafusion-python-embed/src/lib.rs

```diff
@@ -81,9 +81,10 @@ impl PyChartState {
             .map(|el| Ok(depythonize::<ExportUpdateJSON>(el.as_ref(py))?))
             .collect::<PyResult<Vec<_>>>()?;
 
-        let result_updates = self
-            .tokio_runtime
-            .block_on(self.state.update(&self.runtime, updates))?;
+        let result_updates = py.allow_threads(|| {
+            self.tokio_runtime
+                .block_on(self.state.update(&self.runtime, updates))
+        })?;
```

Comment:
This was needed to avoid a deadlock now that the DataFusion datasource may need to acquire the GIL